icu4j/main/common_tests/src/test/java/com/ibm/icu/dev/test/format/MeasureUnitTest.java

```diff
@@ -1348,6 +1429,19 @@ public void TestInvalidIdentifiers() {
 
                 // Compound units not supported in mixed units yet. TODO(CLDR-13701).
                 "kilonewton-meter-and-newton-meter",
+
+                // Invalid units because of invalid constant denominator
+                "meter-per--20--second",
+                "meter-per-1000-1e9-second",
+                "meter-per-1e20-second",
+                "per-1000",
+                "meter-per-1000-1000",
+                "meter-per-1000-second-1000-kilometer",
+                "1000-meter",
+                "meter-1000",
+                "meter-per-1000-1000",
+                "meter-per-1000-second-1000-kilometer",
+                "per-1000-and-per-1000",
```

Comment:
Weren't you going to have a test that verified that the constant came right after the "`-per-`"?  Maybe something like "`meter-per-second-1000`"?

Reply:
sorry, I just saw it after the merge, I am going to add this in a following PR.